shark_turbine/kernel/wave/utils.py

```diff
@@ -334,6 +334,9 @@ def compile_and_invoke(
     if config.get("print_ir_after_all", False):
         flags.append("--mlir-print-ir-after-all")
 
+    if config.get("dump_intermediates", False):
+        flags.append("--iree-hal-dump-executable-intermediates-to=tmp/intermediates")
```

Comment:
It's better to get path from the config, I think, `config["dump_intermediate"] = "some/path"`

Reply:
sure I will change it so that if you specify "dump_intermediates", you also need to specify a path.